### PR TITLE
integrationtests: check for stat output of policy resources

### PR DESCRIPTION
**Pull request by Pothulapati**
This PR adds a new `policy` integration tests suite
to check the output of for the `srv` and `saz` commands and
make sure they have the expected format.

This is majorly needed to make sure things work as expected
with stat cmd output without having to do manual testing.

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

File: test/integration/install_test.go [hunk 1]
```diff
@@ -1131,6 +1131,7 @@ func TestDashboard(t *testing.T) {
 			"dashboard command failed. Expected response [%s] to contain version [%s]",
 			resp, TestHelper.GetVersion())
 	}
+
```
Comment (kleimkuhler) [inline test/integration/install_test.go:1134]:
```suggestion
```

File: test/integration/policy/policy_test.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+		if err != nil {
+			testutil.AnnotatedFatalf(t, "'linkerd inject' command failed",
+				"'linkerd inject' command failed\n%s\n%s", out, stderr)
+		}
+
+		out, err = TestHelper.KubectlApply(out, prefixedNs)
+		if err != nil {
+			testutil.AnnotatedFatalf(t, "failed to update traffic split resource",
+				"failed to update traffic split resource: %s\n %s", err, out)
+		}
```
Comment (kleimkuhler) [inline test/integration/policy/policy_test.go:47]: wrong error message here

File: test/integration/policy/policy_test.go [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+		return fmt.Errorf("No stats found for [%s]", name)
+	}
+
+	/*
+		expectedSuccessRate := "100.00%"
+		if stat.Success != expectedSuccessRate {
+			return fmt.Errorf("Expected success rate [%s] for [%s], got [%s]",
+				expectedSuccessRate, name, stat.Success)
+		}
+	*/
```
Comment (kleimkuhler) [inline test/integration/policy/policy_test.go:165]: looks like this can be removed?

File: test/integration/policy/policy_test.go [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+	})
+}
+
+func validateAuthzRows(name string, rowStats map[string]*testutil.RowStat, isServer bool) error {
+	stat, ok := rowStats[name]
+	if !ok {
+		return fmt.Errorf("No stats found for [%s]", name)
+	}
+
+	/*
```
Comment (adleong) [inline test/integration/policy/policy_test.go:159]: remove this or add a comment explaining it

**Review by adleong — APPROVED** [review]

File: test/integration/policy/policy_test.go [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+		expectedSuccessRate := "100.00%"
+		if stat.Success != expectedSuccessRate {
+			return fmt.Errorf("Expected success rate [%s] for [%s], got [%s]",
+				expectedSuccessRate, name, stat.Success)
+		}
+	*/
+
+	if isServer {
+		if !strings.HasSuffix(stat.UnauthorizedRPS, "rps") {
+			return fmt.Errorf("Unexpected UnauthorizedRPS for [%s], got [%s]",
```
Comment (kleimkuhler) [inline test/integration/policy/policy_test.go:169]:
```suggestion
			return fmt.Errorf("Unexpected Unauthorized RPS for [%s], got [%s]",
```

**Review by kleimkuhler — APPROVED** [review]
Left some trivial comments, but otherwise looks good.

**Pull request merged**